### PR TITLE
Remove vexriscv support.

### PR DESCRIPTION
**Pull request by khrs**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Removed
+- Removed vexriscv-specific registers
```
Comment (Disasm) [inline CHANGELOG.md:11]:
```suggestion
- vexriscv-specific registers moved to the `vexriscv` crate
```
I think this describes the intention better :)

**Comment by almindor** [comment]
CI and merge conflicts need fixing, otherwise tho I'm leaning in support of this

**Comment by Disasm** [comment]
Since we have a decision on this (see https://github.com/xobs/vexriscv-rust/issues/2), I think we can merge this PR. However, there are duplicated commits in this PR. @khrs, could you fix this to make the commit set linear and based on the latest master?

**Comment by Disasm** [comment]
Thank you!
I fixed the commit tree so that it consists of a single commit you are trying to merge.

**Pull request merged**